sdk/textanalytics/azure-ai-textanalytics/tests/testcase.py

```diff
@@ -87,14 +87,22 @@ def __init__(self):
         )
 
     def create_resource(self, name, **kwargs):
-        text_analytics_account = TextAnalyticsTest._TEXT_ANALYTICS_ACCOUNT
-
-        return {
-            'location': 'westus2',
-            'resource_group': TextAnalyticsTest._RESOURCE_GROUP,
-            'text_analytics_account': text_analytics_account,
-            'text_analytics_account_key': TextAnalyticsTest._TEXT_ANALYTICS_KEY,
-        }
+        if self.is_live:
+            text_analytics_account = TextAnalyticsTest._TEXT_ANALYTICS_ACCOUNT
+
+            return {
+                'location': 'westus2',
+                'resource_group': TextAnalyticsTest._RESOURCE_GROUP,
+                'text_analytics_account': text_analytics_account,
+                'text_analytics_account_key': TextAnalyticsTest._TEXT_ANALYTICS_KEY,
+            }
+        else:
+            return {
+                'location': 'westus2',
+                'resource_group': TextAnalyticsTest._RESOURCE_GROUP,
+                'text_analytics_account': "https://westus2.ppe.cognitiveservices.azure.com",
+                'text_analytics_account_key': "fake_key",
+            }
```

Comment:
do the patch for the canary credentials environment variables here in `create_resource` instead of below in the fixture